### PR TITLE
add prediction layer for  NeuMF

### PR DESCRIPTION
**Pull request by LeeJZh**
there should be a dense layer instead of sum operation from neumf to prediction the final score as describe in the text.

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_recommender-systems/neumf.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ class NeuMF(nn.Block):
         for num_hiddens in nums_hiddens:
             self.mlp.add(nn.Dense(num_hiddens, activation='relu',
                                   use_bias=True))
+        self.prediction_layer = nn.Dense(1, activation='sigmoid', use_biad=False)
```
Comment (astonzhang) [inline chapter_recommender-systems/neumf.md:65]: use_bias

**Comment by astonzhang** [comment]
@cheungdaven Can you take a look?

**Comment by cheungdaven** [comment]
@LeeJZh 
Thank you for the revision. 
You have a typo in the code "use bias", could you fix that?

**Comment by astonzhang** [comment]
Thanks. Will merge and fix.

**Pull request merged**